bin/smack.py

```diff
@@ -140,6 +198,14 @@ def arguments():
   verifier_group.add_argument('--smackd', action="store_true", default=False,
     help='generate JSON-format output for SMACKd')
 
+  svcomp_group = parser.add_argument_group('svcomp options')
+
+  svcomp_group.add_argument('--svcomp', action="store_true", default=False,
+    help='enter svcomp mode')
+
+  svcomp_group.add_argument('--error-witness', metavar='FILE', default=None, type=str, 
```

Comment:
I like this idea because I can't really imagine a scenario when we would need both a `.txt` trace output and a `.graphML` error witness trace.

With that said, however, I dislike this idea because the purpose of `--error-witness` is clear from context, and it is self documenting.  If we get rid of `--error-witness` in lieu of `--trace-file` whose output format is dependent on whether the `--svcomp` mode is set, then users will have to be aware that to get an error witness, you must be in `--svcomp` mode, and you must use the `--trace-file` flag.

Getting rid of `--error-witness` would also mean that there is no way to get a `.txt` error trace file when using `--svcomp` mode.  This is probably fine, since you can always pipe output to file, but it should at least be considered when making a decision here.

Finally, I think it is conceivable that we might want to use error-witness checking outside of the context of svcomp, to 'verify' our verifications.  For example, if we have a non-svcomp program that is giving an error, but we believe it should be bug free, we could use error witness checking to confirm or dispel our suspicions.

I really am fine, either way - I don't have a strong opinion about this.  However, I wanted to point out the few considerations that came to mind on this.

Can we come to a consensus on this, and I'll be happy to implement whichever option we converge on?